app/src/NativeBase/Components/ProfileButtons.tsx

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
I'm a bit confused as to why we need this component, when it basically just returns a button?  It doesn't do anything else apart from passing on various props -- so is there a reason not to just directly use a `<Button />` instead and remove this component?